DEVELOPER_GUIDE.md

```diff
@@ -303,6 +303,10 @@ The artifact will include extension settings for the sample Hello World extensio
   opensearchPort: 9200
 ```
 
+You can optionally add `routeNamePrefix:` as a value to the yml. This setting allows you to prefix all you registered NamedRoute names.
+The value must be alphanumeric and can contain `:`, `/`, `*`, `_` in the name.
+
```

Comment:
```suggestion
```

Reply:
there were two new-lines, removed the one below your suggestion.